src/translator/response_builder.cpp

```diff
@@ -1,27 +1,28 @@
 #include "response_builder.h"
 #include "response_options.h"
+#include "quality_estimator.h"
 
 namespace marian {
 namespace bergamot {
 
+
 void ResponseBuilder::buildQualityScores(Histories &histories,
                                          Response &response) {
-  std::vector<Quality> qualityScores;
+  QualityEstimator model(qualityEstimator_->begin());
```

Comment:
Don't construct your class every request.  Member variable of `Service`.

Reply:
Done: 064cfe3